upgrading/automated_upgrades.adoc

```diff
@@ -782,25 +782,22 @@ other nodes in their class (`app` versus `infra`), one at a time.
 .. Run `oc get daemonset` to verify the label found under `NODE-SELECTOR`. The
 default value is `storagenode=glusterfs`.
 
-.. Remove the daemonset label from the node:
+.. Add a label (for example, `type=upgrade`) to the node you want to upgrade.
 +
+[NOTE]
+====
+Use the *_upgrade_nodes.yaml_* playbook:
```

Comment:
I haven't worked enough with the 3.x docs to know if the guidance differs there, but I thought file names just went in backticks (instead of bold and italic).

Reply:
Good question, but let me tell you why I did it the way I did... I copied the format from previous references in the file. I hope that is ok. I thought it best to use the same format that was a couple of hundred lines above it.

I wonder if it was some time after 3.9 that the format for references changed?

Reply:
The format for file names changed in the 4.x stream, and these are right for 3.x.